doc/user-guide/usage.md

```diff
@@ -0,0 +1,380 @@
+topolvm-node-pbcth                    4/4     Running   0          10m
+vg-manager-f979f                      1/1     Running   0          10m
+```
+When all pods are ready, the LVM physical volumes and volume groups should be
+created on the host.
+
+``` console
+sh-4.4# pvs
+  PV           VG  Fmt  Attr PSize  PFree 
+  /dev/nvme0n1 vg1 lvm2 a--  <1.46t <1.46t
+  /dev/nvme1n1 vg1 lvm2 a--  <1.46t <1.46t
+  /dev/nvme2n1 vg1 lvm2 a--  <1.46t <1.46t
+sh-4.4# vgs
+  VG  #PV #LV #SN Attr   VSize  VFree 
+  vg1   3   0   0 wz--n- <4.37t <4.37t
+```
+
+Confirm that the following resources have been created on the cluster:
+
+- When LVMCluster CR is deployed which contains details of all volume groups that
+  needs to be created across multiple nodes, two supporting Custom Resources
+  are created by LVMO
+- LVMVolumeGroup: CR created and managed by LVMO which tracks individual volume
+  groups across multiple nodes
+``` json
+# kubectl get lvmvolumegroup test -ojson
+{
+    "apiVersion": "lvm.topolvm.io/v1alpha1",
+    "kind": "LVMVolumeGroup",
+    "metadata": {
+        "creationTimestamp": "2022-02-02T05:16:42Z",
+        "generation": 1,
+        "name": "test",
+        "namespace": "lvm-operator-system",
+        "resourceVersion": "17242461",
+        "uid": "88e8ad7d-1544-41fb-9a8e-12b1a66ab157"
+    },
+    "spec": {}
+}
+```
+- LVMVolumeGroupNodeStatus: CR created and managed by VG Manager which tracks
+  node status corresponding to multiple volume groups
+``` json
+# kubectl get lvmvolumegroupnodestatuses.lvm.topolvm.io kube-node -ojson
+{
+    "apiVersion": "lvm.topolvm.io/v1alpha1",
+    "kind": "LVMVolumeGroupNodeStatus",
+    "metadata": {
+        "creationTimestamp": "2022-02-02T05:17:59Z",
+        "generation": 1,
+        "name": "kube-node",
+        "namespace": "lvm-operator-system",
+        "resourceVersion": "17242882",
+        "uid": "292de9bb-3a9b-4ee8-946a-9b587986dafd"
+    },
+    "spec": {
+        "nodeStatus": [
+            {
+                "devices": [
+                    "/dev/nvme0n1",
+                    "/dev/nvme1n1",
+                    "/dev/nvme2n1"
+                ],
+                "name": "test",
+                "status": "Ready"
+            }
+        ]
+    }
+}
+```
+- A storage class which matches names (prefixed with `topolvm-`) of
+  deviceClasses in LVMCluster CR are created as below
+``` console
+# kubectl get storageclass
+NAME          PROVISIONER          RECLAIMPOLICY   VOLUMEBINDINGMODE      ALLOWVOLUMEEXPANSION   AGE
+topolvm-vg1   topolvm.cybozu.com   Delete          WaitForFirstConsumer   true                   31m
+```
+
+Note:
+- Reconciling multiple LVMCluster CRs by LVMO is not supported
+- Custom resources LVMVolumeGroup and LVMVolumeGroupNodeStatus are managed by
+  LVMO and users should not edit them.
+
+## Deploying PVC and App Pod
+
+- A successful reconciliation of LVMCluster CR will setup all the underlying
```

Comment:
Can the user see some status on the CR to know it's ready? I'd recommend helping them see the status instead of talking about the reconcile. In my experience, users just don't generally understand what reconcile means.

Reply:
- ack, maybe I can provide a listing of LVMCluster CR after successful reconciliation for easy ref here